frontend/src/pages/articles/my-articles.tsx

```diff
@@ -84,28 +112,41 @@ const MyArticles: React.FC = () => {
   return (
     <>
       <div className={styles.articlesWrapper}>
-        <InfiniteScroll
-          hasMore={hasMore}
-          className={styles.articles}
-          dataLength={articles.length}
-          fetchData={handleLoadArticles}
+        {articles.length ? (
```

Comment:
Boolean(articles.length)
```suggestion
        {Boolean(articles.length) ? (
```

Reply:
Can I change eslint configuration?
![image](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/assets/84803938/3880d041-0649-4fc0-ba0a-2dc84d0588b0)

Reply:
Yes